xml/favicons.xml

```diff
@@ -196,6 +196,14 @@
     <param pos="0" name="service.certainty" value="0.5"/>
   </fingerprint>
 
+  <fingerprint pattern="^53317933c27890ae9218697ecc0e97d9$">
+    <description>SolarWinds Orion</description>
+    <example>53317933c27890ae9218697ecc0e97d9</example>
+    <param pos="0" name="service.vendor" value="SolarWinds"/>
+    <param pos="0" name="service.product" value="Orion"/>
+    <param pos="0" name="service.certainty" value="0.5"/>
```

Comment:
Should we add the CPE for both of these fingerprints? 
`<param pos="0" name="service.cpe23" value="cpe:/a:solarwinds:orion_platform:-"/>`

Reply:
I've updated both with the CPEs.